code/game/objects/items/devices/dociler.dm

```diff
@@ -53,17 +55,36 @@
 			H.friends += user
 
 	L.desc += "<br><span class='notice'>It looks especially docile.</span>"
-	var/name = input(user, "Would you like to rename \the [L]?", "Dociler", L.name) as text
-	if(length(name))
-		L.real_name = name
-		L.name = name
+
+	if(can_rename)
+		var/name = input(user, "Would you like to rename \the [L]?", "Dociler", L.name) as text
+		if(length(name))
+			L.real_name = name
+			L.name = name
 
 	loaded = 0
 	icon_state = "animal_tagger0"
-	addtimer(CALLBACK(src, .proc/do_recharge), 5 MINUTES)
-
+	addtimer(CALLBACK(src, .proc/do_recharge), cooldown)
 
 /obj/item/device/dociler/proc/do_recharge()
 	loaded = 1
 	icon_state = "animal_tagger1"
-	src.visible_message("\The [src] beeps, refilling itself.")
\ No newline at end of file
+	src.visible_message("\The [src] beeps, refilling itself.")
+
+/obj/item/device/dociler/light
```

Comment:
No, the station is not really supossed to have access to a dociler without a merchant, and I do not see how it makes sense for the xenoarcheologist, that has nothing to do with living animals, to have access to this. A gun to protect himself against hostile fauna is fine, but the dociler is not.